modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/renderer/ObjectEntryBasicInfoItemRenderer.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.web.internal.info.item.renderer;
+
+import com.liferay.info.item.renderer.InfoItemRenderer;
+import com.liferay.object.model.ObjectDefinition;
+import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.service.ObjectDefinitionLocalService;
+import com.liferay.object.service.ObjectEntryLocalService;
+import com.liferay.object.web.internal.constants.ObjectWebKeys;
+import com.liferay.portal.kernel.language.LanguageUtil;
+
+import java.io.Serializable;
+
+import java.util.Locale;
+import java.util.Map;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Jorge Ferrer
+ */
+@Component(
+	enabled = false, property = "service.ranking:Integer=100",
+	service = InfoItemRenderer.class
+)
+public class ObjectEntryBasicInfoItemRenderer
```

Comment:
Just for the record, once this class is deployed, it can be enabled from the Gogo shell by executing the following command:
`
scr:enable com.liferay.object.web.internal.info.item.renderer.ObjectEntryBasicInfoItemRenderer
`

Reply:
Ok, thank you!